starters/prisma/src/app/pages/Home.tsx

```diff
@@ -0,0 +1,8 @@
+
+export function Home() {
+  return (
+    <div>
+      <h1>Hello World</h1>
+    </div>
+  );
+}
```

Comment:
If I'm remembering right, I think the loom used the `db` here. Just to be sure, did we intentionally not include the same here?